docs/developer-guide/development-environment.md

```diff
@@ -55,9 +55,7 @@ Running the tests is the best way to ensure you have correctly set up your devel
 npm test
 ```
 
-The testing takes a few seconds to complete. If any tests fail, that likely means one or more parts of the environment setup didn't complete correctly. The upstream tests always pass.
-
-
+The testing takes about 2 minutes to complete. If any tests fail, that likely means one or more parts of the environment setup didn't complete correctly. The upstream tests always pass.
```

Comment:
Can we maybe change this to `a few minutes to complete` instead? That way we don't have to keep updating this should they change in the future!

Reply:
Sure. Good idea. Will do.